### ISSUE TITLE
ScheduledQuartzJob with cluster=true does not work

### ISSUE THREAD

**Issue by tgeens** [xenit-eu/dynamic-extensions-for-alfresco#226]
## I'm submitting a ...   (check one with "x")
```
[x] bug report
[ ] feature request
[ ] question
```

## Expected Behavior
Expect it to be invoked (on 1 of the members of a cluster)

## Current Behavior
```
[DefaultScheduler_QuartzSchedulerThread] An error occured instantiating job to be executed. job= 'testing.integration-test-hello'
org.quartz.SchedulerException: Job instantiation failed [See nested exception: java.lang.InstantiationException]
	at org.springframework.scheduling.quartz.AdaptableJobFactory.newJob(AdaptableJobFactory.java:59)
	at org.quartz.core.JobRunShell.initialize(JobRunShell.java:140)
	at org.quartz.core.QuartzSchedulerThread.run(QuartzSchedulerThread.java:365)
Caused by: java.lang.InstantiationException
	at sun.reflect.InstantiationExceptionConstructorAccessorImpl.newInstance(InstantiationExceptionConstructorAccessorImpl.java:48)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at org.springframework.scheduling.quartz.AdaptableJobFactory.createJobInstance(AdaptableJobFactory.java:77)
	at org.springframework.scheduling.quartz.SpringBeanJobFactory.createJobInstance(SpringBeanJobFactory.java:77)
	at org.alfresco.repo.scheduler.AlfrescoJobFactory.createJobInstance(AlfrescoJobFactory.java:52)
	at org.springframework.scheduling.quartz.AdaptableJobFactory.newJob(AdaptableJobFactory.java:55)
	... 2 more
```

## Steps to Reproduce (for bugs)
1. Create an `@QuartzJobSchedule(name="foo", cluster="true", cron = ...)

## Your Environment

* Alfresco version used: 5.2.4
* DE version used: 2.0-SNAPSHOT

**Comment by tgeens** [comment]
It tries to create an instance of the abstract class `AbstractScheduledLockedJob`. That won't work.

**Comment by kerkhofsd** [comment]
Fixed by #227
